manifests/bosh-manifest/deployments/040-collectd.yml

```diff
@@ -0,0 +1,56 @@
+---
+releases:
+- name: collectd
+  sha1: 5e5c196f9c95fe12eb17bba0f40fdaecaddd6424
+  url: https://github.com/alphagov/paas-collectd-boshrelease/releases/download/0.4/collectd-0.4.tgz
+
+jobs:
+- name: bosh
+  templates:
+  - {name: collectd, release: collectd}
+
+properties:
+  collectd:
+    interval: 10
+    config: |
```

Comment:
Right now they are the same, except for comments. I was looking how to make it the same (meaning, included from single place) for all of them but it is non-trivial. In cf-manifest, this is a runtime addon. We'd have to parse it out from that file to include it into bosh - in a different location. Concourse manifest is not even composed from multiple files, so that would require another way again. Altogether this looks like a big piece of work that we might not want to do for this story. Second question is if we really want to have all the settings the same. E.g. if we need to deploy a collectd fix for concourse (0.4), there's no need to re-deploy bosh as well and do rolling update of whole CF deployment....